assets/styles/all-stylesheets.css

```diff
@@ -0,0 +1,12 @@
+@import 'units.css';
+@import 'grid.css';
+@import 'sidebar.css';
+@import 'text-columns.css';
+@import 'fonts.css';
+@import 'colors.css';
+@import 'elements.css';
+@import 'animation.css';
+@import 'icons.css';
+@import 'mlh-trust-badge.css';
+
+/* @import 'schedule.styl'; */
```

Comment:
Could you explain what this commented out code is for?

Reply:
The schedule.styl file is for the schedule page, which isn't part of the static site and so hasn't been migrated yet. I'd like to keep it in the manifest just to record it's there, then get rid of it when I migrate the schedule page.

Reply:
Please could you add in a comment explaining this?

Reply:
This should now be resolved